exportoptions/appstore_options.go

```diff
@@ -26,9 +27,25 @@ type AppStoreOptionsModel struct {
 	TestFlightInternalTestingOnly bool
 }
 
-// NewAppStoreOptions ...
+// NewAppStoreOptions sets "app-store" as the export method
+// deprecated: use NewAppStoreConnectOptions instead
 func NewAppStoreOptions() AppStoreOptionsModel {
 	return AppStoreOptionsModel{
```

Comment:
this could be `NewAppStoreConnectOptions(MethodAppStore)`, right?

Reply:
Yes it is the same. It could be removed, but there is a few code using it: https://github.com/search?q=org%3Abitrise-steplib%20NewAppStoreOptions&type=code

Reply:
Yeah, that make sense, I ment the function could `return NewAppStoreConnectOptions(MethodAppStore)`, instead of creating a new AppStoreOptionsModel by hand.